pygus/gus/inputs/site.json

```diff
@@ -1,8 +1,8 @@
 {
-    "area_total_in_m2":1000,
-    "area_impervious_in_m2":500,
-    "area_pervious_in_m2": 500,
-    "area_tree_density_per_hectare": 400,
+    "total_m2":1000,
+    "impervious_m2":500,
+    "pervious_m2": 500,
+    "tree_density_per_ha": 400,
     "weather": {
         "growth_season_mean": 200,
         "growth_season_var": 7
```

Comment:
This may also be important. I've changed the site JSON names to reduce them a bit. I guess this affects any custom site config files that have been made, but I don't know if there have been (m)any??